### ISSUE TITLE
Update failed, loop to logo

### ISSUE THREAD

**Issue by Galiwyx** [hoobs-org/HOOBS#622]
HEllo,

3 days ago I tried to update my hoobs installation (raspberry version). It seems update failed, since the update hoobs.local display the same screen : 

![image](https://user-images.githubusercontent.com/206181/81486281-f34be680-9253-11ea-95f2-42af2291ef41.png)

I don't know what to do.

Thank you :)

**Comment by askovi** [comment]
Please login via ssh with user: hoobs and password: hoobsadmin

Then run this command to get the logs:

`sudo hoobs service log`

**Comment by Galiwyx** [comment]
Hello,
I tried but it retrieves an error : 

![image](https://user-images.githubusercontent.com/206181/81486461-41151e80-9255-11ea-97d9-1619a12e5c1b.png)

**Comment by askovi** [comment]
Ok try `sudo journalctl -f -u hoobs`

**Comment by Galiwyx** [comment]
Okay

![image](https://user-images.githubusercontent.com/206181/81486573-edef9b80-9255-11ea-94a4-f9bb3a5f586d.png)

**Comment by mkellsy** [comment]
We have a fix for this in version 3.2.4

**Comment by mkellsy** [comment]
You can run these commands to get this working
```
sudo systemctl disable nginx
sudo setcap CAP_NET_BIND_SERVICE=+eip /usr/local/lib/node_modules/@hoobs/hoobs/bin/hoobs
sudo setcap CAP_NET_BIND_SERVICE=+eip /usr/local/bin/node
sudo reboot
```
